test/e2e/specs/wp-my-home-support-search-spec.js

```diff
@@ -78,48 +61,59 @@ describe( `[${ host }] My Home "Get help" support search card: (${ screenSize })
-
-		// Once removed we can assert is is invisible.
-		const isToggleVisible = await inlineHelpPopoverComponent.isToggleVisible();
-		assert.equal(
-			isToggleVisible,
-			false,
-			'Inline Help support search was not correctly hidden on Home page.'
-		);
-	} );
-
 	step( 'Displays contextual search results by default', async function () {
 		supportSearchComponent = await SupportSearchComponent.Expect( driver );
-		const resultsCount = await supportSearchComponent.getSearchResultsCount();
-		assert.equal( resultsCount, 5, 'There are no contextual results displayed' );
+		const resultsCount = await supportSearchComponent.getDefaultResultsCount();
+		assert.equal( resultsCount, 5, 'The default results are displayed' );
 	} );
 
 	step( 'Returns search results for valid search query', async function () {
-		await supportSearchComponent.searchFor( 'Podcast' );
-		const resultsCount = await supportSearchComponent.getSearchResultsCount();
+		await supportSearchComponent.searchFor( 'Domain' );
+
+		const searchResultsCount = await supportSearchComponent.getSearchResultsCount();
+
+		const adminResultsCount = await supportSearchComponent.getAdminSearchResultsCount();
+
+		const defaultResults = await supportSearchComponent.waitForDefaultResultsNotToBePresent();
+
+		assert.equal( defaultResults, true, 'The default results are not displayed.' );
 
 		assert.equal(
-			resultsCount <= 5,
+			searchResultsCount <= 5,
 			true,
-			`Too many search results displayed. Should be less than or equal to 5 (was ${ resultsCount }).`
+			`Too many search results displayed. Should be less than or equal to 5.`
 		);
 		assert.equal(
-			resultsCount >= 1,
+			searchResultsCount >= 1,
 			true,
-			`Too few search results displayed. Should be more than or equal to 1 (was ${ resultsCount }).`
+			`Too few search results displayed. Should be more than or equal to 1.`
+		);
+		assert.equal(
+			adminResultsCount <= 3,
+			true,
+			`Too many admin results displayed. Should be less than or equal to 3.`
+		);
+		assert.equal(
+			adminResultsCount >= 1,
+			true,
+			`Too few admin results displayed. Should be more than or equal to 1.`
 		);
 	} );
 
 	step( 'Resets search UI to default state when search input is cleared ', async function () {
 		await supportSearchComponent.clearSearchField();
 
-		const resultsCount = await supportSearchComponent.getSearchResultsCount();
+		const searchResults = await supportSearchComponent.waitForSearchResultsNotToBePresent();
+
+		const adminResults = await supportSearchComponent.waitForAdminResultsNotToBePresent();
 
-		assert.equal( resultsCount, 5, 'There are no contextual results displayed' );
+		const defaultResultsCount = await supportSearchComponent.getDefaultResultsCount();
+
+		assert.equal( searchResults, true, 'The API Search Results are not displayed.' );
+
+		assert.equal( adminResults, true, 'The Admin Results are not displayed.' );
+
+		assert.equal( defaultResultsCount, 5, 'The default results displayed' );
```

Comment:
Minor: should we be consistent with `Results` vs `results`?

Reply:
I went through and tried to make the naming of everything more consistent.